packages/daemon/src/mutex.js

```diff
@@ -21,7 +28,7 @@ export const makeMutex = () => {
     enqueue: async asyncFn => {
       await lock();
       try {
-        return await asyncFn();
+        return await (asyncFn ? asyncFn() : null);
```

Comment:
Better to default the ayncFn to noop.

Reply:
I also don’t see this getting used. Is it necessary?

Reply:
It is used at the top of the added `cancelValue` method. I did change the implementation to use a default noop function, however.